lib/inheritance_integer_type/version.rb

```diff
@@ -1,3 +1,3 @@
 module InheritanceIntegerType
-  VERSION = "0.1.3"
+  VERSION = "0.2.0"
```

Comment:
why the bump to `0.2.0` instead of say `0.1.4`?

Reply:
I thought it was because it is a more major change since this is dropping ruby 2.7 support.

Reply:
Yup what @anthonyoconnorclio said. Ruby 3 vs 2.7  (we can argue semver if you like) I wanted it to be clear that if you want to change ruby versions its a clear minor version bump rather and a incremental patch level bump. 

Easier to clarify in the docs too, IMO.